app/controllers/cuisine_controller.rb

```diff
@@ -0,0 +1,9 @@
+class CuisineController < ApplicationController
+  def index
+    render json: Cuisine.all
+  end
+
+  def create
+    render json: Cuisine.create(name: params.require(:name))
```

Comment:
here you've used `create` for for `Restaurant` you used `create!`. you should probably decide on one (hint, use `create!`)